static/panes/compiler.js

```diff
@@ -1044,6 +1099,7 @@ Compiler.prototype.initButtons = function (state) {
     this.filters = new Toggles(this.domRoot.find('.filters'), patchOldFilters(state.filters));
 
     this.optButton = this.domRoot.find('.btn.view-optimization');
+    this.flagsButton = false; // We construct this when we populate the Popular Arguments drodown.
```

Comment:
Can we make this `null` please? `false` implies it's a boolean and it seems like it's really an optional dom node, right?

Reply:
I actually can just use domRoot.find now that it's specified in the .pug

Reply:
I think it's ready; this bit is fixed I just had the one comment I couldn't figure out